components/CardDropdownComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { styled } from "styled-components";
+import { useModalSetting } from "../hooks/useModalSetting";
+import CommonModalComponent from "./CommonModalComponent";
+
+type props = {
+  show: boolean;
+  url: string;
+};
+
+const CardDropdownComponent = ({ show, url }: props) => {
```

Comment:
-Component는 제거해주시는게 좋습니다!
```suggestion
const CardDropdown = ({ show, url }: props) => {
```